README.md

```diff
@@ -1,2 +1,23 @@
-# HW01
-Edit the readme
+---
+title: "Homework 1"
+author: "Ben Wang"
+---
+
+### bwang5@uchicago.edu
+
+Hello all!
+My name is Ben, I'm an incoming first year student in the **University of Chicago** *Neurobiology* program [link here](https://neuroscience.uchicago.edu/). I am interested in studying the molecular mechanisms behind automatic functions such as breathing, pain and prioprioception. I have no prior experience in coding but have always been interested in learning!
+
+I am originally from Winnipeg, Canada, and came to the US for college. I majored in molecular and cellular biology and minored in music.
```

Comment:
Wow it's nice to hear that you minored in music! I learned to play the guitar for a couple of months, in the most unofficial way :), and I sing a lot. I've been singing in an A cappella group in college for 4 years. Maybe you guys are quite familiar with A cappella, I get extremely excited when everyone sings along without any instruments!